lib/ClangImporter/ClangImporter.cpp

```diff
@@ -731,6 +810,7 @@ ClangImporter::create(ASTContext &ctx,
 
   if (llvm::sys::path::extension(importerOpts.BridgingHeader).endswith(
         PCH_EXTENSION)) {
+    importer->Impl.setSinglePCHImport(importerOpts.BridgingHeader);
```

Comment:
If the PCH name gets passed to the action, you can drop this. Having only one place call `setSinglePCHImport` feels a lot harder to screw up.